### DISCUSSION TITLE
Delete account

### DISCUSSION THREAD

**Discussion by ghost** [shellhub-io/shellhub#3287] [Q&A]
Hello,

How can I delete my account on https://cloud.shellhub.io/ ? 

I tried to "Delete Namespace" but, I keep in a endless loading.

![Screenshot 2023-11-13 at 18 34 54](https://github.com/shellhub-io/shellhub/assets/40402385/01e8bd91-3f5e-413f-9616-f8e36c01dd16)

How can I do that? I didn't find any email to solve that.

**Comment by heiytor** [comment]
Hi, we're currently investigating this issue. Could you please send the tenant-id of the entity that is trying to delete?

**Reply by ghost** [reply]
Hello @heiytor,

The tenant-id: c96924a6-4375-4713-9028-bfd6803e1df7

I'll try to show an error image.

**Reply by ghost** [reply]
<img width="805" alt="Screenshot 2023-11-14 at 12 23 10" src="https://github.com/shellhub-io/shellhub/assets/40402385/ef42e825-5e7f-46ff-a287-d6f8efa4a077">

This is what's happening when I try to delete. It keeps loading (as the first image) for 3 or 5 minutes. And I get this error.

**Comment by heiytor** [comment]
Hi, @Cahoots2843.

The problem has already been identified and resolved. You should be able to remove the tenant when the next release comes in.

However, if you're looking to exclude the entire account, this is currently not possible through the UI. Nevertheless, you can send an email to contact@shellhub.io from the email registered at ShellHub, requesting the removal.
